stetho/src/main/java/com/facebook/stetho/inspector/protocol/module/CSS.java

```diff
@@ -348,4 +381,18 @@ public PseudoIdMatches() {
     @JsonProperty
     public List<InheritedStyleEntry> inherited;
   }
+
+
+  private static class SetPropertyTextRequest implements JsonRpcResult {
+      @JsonProperty(required = true)
```

Comment:
nit: the indentation of this class is super weird ... 4 spaces indent, and then 2 spaces un-indent at the end? pretty sure this should be 2 and 2